content/blog/2024/oci-manifest-capability.md

```diff
@@ -205,8 +212,7 @@ manifest:
 ]
 ```
 
-The second JSON document is the manifest for the second container, which uses a
-Kubewarden policy as its image:
+The second JSON document is the manifest for the Kubewarden policy:
```

Comment:
I would omit this second JSON document to not confuse the readers and for brevity. Having the paragraph about registries returning an OCI index manifest or OCI image manifest is good, though.